<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStructuredGrid</class>
 <widget class="QDialog" name="CreateStructuredGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Structured Mesh</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="elementTypeBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Element type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="3" column="0">
       <widget class="QRadioButton" name="prismButton">
        <property name="text">
         <string>Prism</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="quadButton">
        <property name="text">
         <string>Quad</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="hexButton">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="lineButton">
        <property name="text">
         <string>Line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="triButton">
        <property name="text">
         <string>Triangle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="paramBox">
     <property name="title">
      <string>Mesh size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="6" column="1">
       <widget class="QLineEdit" name="zElemEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="yElemLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Elements in y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="xLengthEdit">
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="elemExtentButton">
        <property name="text">
         <string>Element extent</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="xElemLabel">
        <property name="text">
         <string>Elements in x</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="zElemLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Elements in z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="meshExtentButton">
        <property name="text">
         <string>Mesh extent</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yLengthLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Length in y</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="yElemEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="xElemEdit">
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="zLengthEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xLengthLabel">
        <property name="text">
         <string>Length in x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="zLengthLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Length in z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="yLengthEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="originBox">
     <property name="title">
      <string>Origin</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="xOriginLabel">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="zOriginLabel">
        <property name="text">
         <string>   z:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="yOriginEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="yOriginLabel">
        <property name="text">
         <string>   y:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="xOriginEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="zOriginEdit">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="meshNameLabel">
     <property name="text">
      <string>Mesh name:</string>
     </property>
     <property name="margin">
      <number>11</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="meshNameEdit">
     <property name="text">
      <string>StructuredMesh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>meshNameEdit</tabstop>
  <tabstop>lineButton</tabstop>
  <tabstop>triButton</tabstop>
  <tabstop>quadButton</tabstop>
  <tabstop>prismButton</tabstop>
  <tabstop>hexButton</tabstop>
  <tabstop>meshExtentButton</tabstop>
  <tabstop>elemExtentButton</tabstop>
  <tabstop>xLengthEdit</tabstop>
  <tabstop>yLengthEdit</tabstop>
  <tabstop>zLengthEdit</tabstop>
  <tabstop>xElemEdit</tabstop>
  <tabstop>yElemEdit</tabstop>
  <tabstop>zElemEdit</tabstop>
  <tabstop>xOriginEdit</tabstop>
  <tabstop>yOriginEdit</tabstop>
  <tabstop>zOriginEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateStructuredGrid</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateStructuredGrid</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
